test/Splunk.OpenTelemetry.AutoInstrumentation.Tests/LoggerTests.cs

```diff
@@ -0,0 +1,37 @@
+// <copyright file="LoggerTests.cs" company="Splunk Inc.">
+// Copyright Splunk Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using Splunk.OpenTelemetry.AutoInstrumentation.Logging;
+
+namespace Splunk.OpenTelemetry.AutoInstrumentation.Tests;
+
+public class LoggerTests
+{
+    [Fact]
+    public void ConstructorDoesNotThrowExceptionWhenReflectionFails()
+    {
+        var exception = Record.Exception(() => new Logger());
+        exception.Should().Be(null);
+    }
+
+    [Fact]
+    public void ImplementationDoesNotThrowExceptionWhenReflectionFails()
+    {
+        ILogger logger = new Logger();
+        var exception = Record.Exception(() => logger.Warning("message"));
+        exception.Should().Be(null);
```

Comment:
```suggestion
        Action act = () => logger.Warning("message");
        act.Should().NotThrow();
```

see: https://fluentassertions.com/exceptions/